config/log4j2.properties

```diff
@@ -15,25 +15,29 @@ appender.json_console.layout.eventEol = true
 appender.rolling.type = RollingFile
 appender.rolling.name = plain_rolling
 appender.rolling.fileName = ${sys:ls.logs}/logstash-${sys:ls.log.format}.log
-appender.rolling.filePattern = ${sys:ls.logs}/logstash-${sys:ls.log.format}-%d{yyyy-MM-dd}.log
+appender.rolling.filePattern = ${sys:ls.logs}/logstash-${sys:ls.log.format}-%d{yyyy-MM-dd}-%i.gz
```

Comment:
keeping the `log` as part of the suffix would be helpful; I'd recommend ending with `%d{yyyy-MM-dd}-%i.log.gz`.

Reply:
Done.